examples/tutorials/07_multiple_policies.py

```diff
@@ -0,0 +1,187 @@
+import torch
+import dataclasses
+import mediapy
+from huggingface_hub import PyTorchModelHubMixin
+from huggingface_hub import ModelCard
+from gpudrive.networks.late_fusion import NeuralNet
+from gpudrive.agents.core import merge_actions
+from gpudrive.env.config import EnvConfig
+from gpudrive.env.env_torch import GPUDriveTorchEnv
+from gpudrive.visualize.utils import img_from_fig
+from gpudrive.env.dataset import SceneDataLoader
+from gpudrive.utils.config import load_config
+import imageio
+import numpy as np
+
+
+def create_policy_masks(env, num_sim_agents=2):
+    policy_mask = torch.zeros_like(env.cont_agent_mask, dtype=torch.int)
+    agent_indices = env.cont_agent_mask.nonzero(as_tuple=True)
+
+    for i, (world_idx, agent_idx) in enumerate(zip(*agent_indices)):
+        policy_mask[world_idx, agent_idx] = (i % num_sim_agents) + 1
+
+    policy_mask = {f'pi_{int(policy.item())}': (policy_mask == policy)
+            for policy in policy_mask.unique() if policy.item() != 0}
+
+
+    policy_world_mask = {
+        world: {f'pi_{p+1}': policy_mask[f'pi_{p+1}'][world] for p in range(num_sim_agents)}
+        for world in range(env.cont_agent_mask.shape[0])
+    }
+    return policy_world_mask
+# Configs model has been trained with
+config = load_config("../../examples/experimental/eval/config/reliable_agents_params")
+
+
+
+max_agents = config.max_controlled_agents
+num_envs = 2
+device = "cpu" # cpu just because we're in a notebook
+NUM_SIM_AGENTS = 2
+
+sim_agent1 = NeuralNet.from_pretrained("daphne-cornelisse/policy_S10_000_02_27")
```

Comment:
Just so that we have a bit more "diversity", could you use the 1000-scene policy for one sim agent and the 10,000 one for the other?